server/src/main/java/com/aws/greengrass/cli/CLIService.java

```diff
@@ -171,11 +175,32 @@ protected void install() {
                 Files.createSymbolicLink(link, binary);
                 logger.atInfo().kv("binary", binary).kv("link", link).log("Set up symlink to CLI binary");
             }
+
+            // authorize pub/sub for the cli
+            authorizePubSubPermission(PUB_SUB_SERVICE_NAME);
+            authorizePubSubPermission(MQTT_PROXY_SERVICE_NAME);
         } catch (IOException | SemverException e) {
             logger.atError().log("Failed to set up symlink to CLI binary", e);
         }
     }
 
+    private void authorizePubSubPermission(String serviceIdentifier) throws IOException {
+        String defaultClientId =
+                USER_CLIENT_ID_PREFIX + Platform.getInstance().lookupCurrentUser().getPrincipalIdentifier();
+        String serviceName = getAuthClientIdentifier(defaultClientId);
```

Comment:
this is not correct and it will not work. Did you test this at all?


The service name should be `CLIService.CLI_SERVICE`